packages/api/lambdas/sf-event-sqs-to-db-records/write-granules.js

```diff
@@ -453,7 +454,8 @@ const _writeGranule = async ({
   await _writeGranuleFiles({
     files,
     granuleCumulusId,
-    granuleId: granule.granuleId,
+    granule,
+    workflowError: error,
```

Comment:
Also does this work right in the underlying method, given `error` is generated from `writeGranules`'s call to `parseException` which does:

```
function parseException(exception) {
  if (isNil(exception)) return {};
  if (isObject(exception)) return exception;
  return {
    Error: 'Unknown Error',
    Cause: exception,
  };
}
```

?

I'll comment on the line above as well.